app/src/main/java/com/github/dedis/student20_pop/model/MeetingEvent.java

```diff
@@ -0,0 +1,51 @@
+package com.github.dedis.student20_pop.model;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class MeetingEvent extends Event {
```

Comment:
I agree with the fact that we have different classes for the different events, but is this supported by the protocol or not yet ?

Reply:
I'm not sure. I need to check that, but I don't think so

Reply:
There is nothing in the protocol for events yet. It needs to be updated